webcompat/api/uploads.py

```diff
@@ -126,16 +118,11 @@ def save(self):
             save_parameters['save_all'] = True
         # unpacking save_parameters
         self.image_object.save(file_dest, **save_parameters)
-        # Creating the thumbnail
-        size = (1024, 1024)
-        self.image_object.thumbnail(size, Image.HAMMING)
```

Comment:
goodbye, you beautiful ham

![home-carousel-1](https://user-images.githubusercontent.com/67283/71830165-cac89b80-306b-11ea-8744-79180281b1f7.jpg)